miso-web/src/main/java/uk/ac/bbsrc/tgac/miso/webapp/util/ExperimentListConfiguration.java

```diff
@@ -78,7 +79,8 @@ public ExperimentListConfiguration(ExperimentService experimentService, LibraryS
         .flatMap(container -> container.getPartitions().stream())//
         .filter(partition -> partition.getPool() != null)//
         .flatMap(partition -> {
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove useless curly braces around statement and then remove useless return keyword [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1602)